test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -272,5 +290,27 @@ public static void positionTestFrame(Frame testFrame, Position position) {
     public static synchronized void addTestFrame(Frame testFrame) {
         frameList.add(testFrame);
     }
+
+    /**
+     * Forcibly pass the test.
+     * <p>The sample usage:
+     * <pre><code>
+     *      PrinterJob pj = PrinterJob.getPrinterJob();
+     *      if (pj == null || pj.getPrintService() == null) {
+     *          System.out.println(""Printer not configured or available.");
+     *          PassFailJFrame.forcePass();
+     *      }
+     * </code></pre>
+     */
+    public static void forcePass() {
+        latch.countDown();
+    }
+
+    /**
+     *  Forcibly pass the test.
+     */
```

Comment:
you mean fail.

Reply:
Yes, I was meant to fail. Let me correct it.
Thanks! for catching. I had fixed this but mostly forget to commit.